app/javascript/leaflet/viewers/iiif.js

```diff
@@ -0,0 +1,36 @@
+import { map, tileLayer, CRS } from "leaflet";
+import "leaflet-iiif";
+import LeafletViewerBase from "./base.js";
+
+export default class LeafletViewerIiif extends LeafletViewerBase {
+  constructor(el) {
+    super(el);
+    this.onLoad();
+  }
+
+  onLoad() {
+    this.adjustLayout();
+
+    this.map = map(this.element, {
+      center: [0, 0],
+      crs: CRS.Simple,
+      zoom: 0,
+    });
+
+    this.iiifLayer = tileLayer.iiif(this.data.url).addTo(this.map);
+    this.addControls();
```

Comment:
Just leaving a note that 20 and 21 are switched in the original code but 99.999% sure it doesn't matter.